### ISSUE TITLE
White screen after loading roms from alternate "games" directory

### ISSUE THREAD

**Issue by PronkMeister** [MiSTer-devel/NeoGeo_MiSTer#133]
When replacing the "/media/fat/games" directory with one mounted from CIFS storage, the NeoGeo core loads the roms but does not boot the BIOS. Instead it only shows a white screen.
To replicate this issue, copy the entire contents of the "games" directory to a CIFS share.
Rename the the games directory to "games_old" or something similar.
Then mount the CIFS share containing only the "games" directory.
Every core will work fine, except the NeoGeo core.
This will present the white screen after loading the selected game roms and stay like this.

**Comment by sorgelig** [comment]
Remote storage may or may not work. Basically it's not a standard way to use MiSTer.

**Comment by PronkMeister** [comment]
While that is absolutely true, the fact remains that this only happens with the NeoGeo core.
Which is why I submitted this as a bug. The roms load just fine, which leads me to believe there is an actual bug in the core.

**Comment by PronkMeister** [comment]
It might help, but this is my cifs_mount.ini configuration:
`SERVER="192.168.1.51"
SHARE="MiSTer"
USERNAME="admin"
PASSWORD="******"
ADDITIONAL_MOUNT_OPTIONS="vers=2.0"
MOUNT_AT_BOOT="true"`

**Comment by sorgelig** [comment]
I'm not using CIFS so i can't help you.

**Comment by PronkMeister** [comment]
Ok, well then this is for the one who can fix this, or someone capable who thinks this is relevant information. 
To add further information to the bug, if the "games" directory is left in tact and the CIFS share contains the folder "games" but renamed to something else like "games-lan" the core will load everything correctly when you select roms through the "games-lan\NeoGeo" directory.

**Comment by misteraddons** [comment]
I'm able to load games over CIFS just fine. Double check your configuration. I have all the core files on my micro SD and the games on my CIFS mounted "games" folder. Nothing else required.

**Comment by nsknyc** [comment]
I used the update_all.sh and I'm getting the same issue. The difference is that my roms and bios are in the sd. I also moved them into mister through filzezilla. I remember reading something on reddit about filezilla being the issue, so I decided to delete the unibios, and rom folder (all which I had transfered over). Then just mounted the sd on my reader and moved them there. Oh before I deleted them I did notice that the date on the files were for the next day. Not sure if that's the issue. I'm new to mister, but I believe that linux does all the house cleaning perhaps thats somewhere to start?
